react-native/setup/setup-and-deploy-new-project-to-staging.md

```diff
@@ -2,6 +2,7 @@
 
 ## Prerequisites
 - [ ] Have you entire environment setup
+- Generating a new SSH key and adding it to the ssh-agent: https://help.github.com/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent/
```

Comment:
Try to use the markdown `[using this guide](https://help.github.com/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent/` as it would improve the readability.